packages/core/backend/sirius-components-collaborative/src/main/resources/schema/omnibox.graphqls

```diff
@@ -0,0 +1,25 @@
+extend type Viewer {
+    omniboxCommands(contextEntries: [OmniboxContextEntry!]!, query: String!): OmniboxCommandsConnection!
+}
+input OmniboxContextEntry {
+    id: ID!
+    label: String!
```

Comment:
The label can be removed, it will be useful on the client, not the backend

Reply:
Done

Reply:
Done